packages/overlay/stories/overlay-element.stories.ts

```diff
@@ -152,6 +153,37 @@ longpress.args = {
     type: 'auto',
 };
 
+/**
+ * Proxy for fully encapsulated overlay containers that need to
+ * pass `focus` into a shadow child element.
+ */
+export const receivesFocus = ({
+    interaction,
+    open,
+    placement,
+    receivesFocus,
```

Comment:
can we write something like `receivesFocus='auto'` as a default value if it doesn't receive any value instead of adding an OR in the .receivesFocus attr

Reply:
Updated!